typing/ctype.ml

```diff
@@ -151,6 +151,18 @@ let global_level = s_ref 1
 let saved_level = s_ref []
 
 let get_current_level () = !current_level
+(* Old API
+val init_def: int -> unit
+        (* Set the initial variable level *)
+val begin_def: unit -> unit
+        (* Raise the variable level by one at the beginning of a definition. *)
+val begin_class_def: unit -> unit
+        (* Raise the current level not touching the nongen level *)
+val raise_nongen_level: unit -> unit
+        (* Raise the nongen level to the current level *)
+val end_def: unit -> unit
+        (* Lower the variable level by one at the end of a definition *)
+*)
```

Comment:
Why are we keeping the old API at all, what is the benefit of keeping this comment around?

Reply:
I removed it. Indeed, it is not very informative here anyway.